fixed broken link (#13479)
docker/docker.github.io: subscription/add-seats.md

5. The **Billing** tab displays the number of seats currently in use.

> 6. Navigate to the **Members** tab to add new members. For more information, see [Add a member to a team](../orgs/#add-a-member-to-a-team).

Reviewer: I think it was broken because its in the "docker-hub" directory (not because of the .md suffix)

```suggestion
6. Navigate to the **Members** tab to add new members. For more information, see [Add a member to a team](../docker-hub/org.md#add-a-member-to-a-team).
```

> 6. Navigate to the **Members** tab to add new members. For more information, see [Add a member to a team](../docker-hub/orgs.md#add-a-member-to-a-tea).

Reviewer: think you missed an m at the end 😁

```suggestion
6. Navigate to the **Members** tab to add new members. For more information, see [Add a member to a team](../docker-hub/orgs.md#add-a-member-to-a-team).
```

5. The **Billing** tab displays the number of seats currently in use.

> 6. Navigate to the **Members** tab to add new members. For more information, see [Add a member to a team](../docker-hub/orgs.md#add-a-member-to-a-tea).

Reviewer: same here 😇

```suggestion
6. Navigate to the **Members** tab to add new members. For more information, see [Add a member to a team](../docker-hub/orgs.md#add-a-member-to-a-team).
```

Author: @thaJeztah Thank you!! =)
